x-pack/plugins/enterprise_search/server/routes/workplace_search/groups.test.ts

```diff
@@ -150,30 +136,20 @@ describe('groups routes', () => {
 
     beforeEach(() => {
       jest.clearAllMocks();
-    });
-
-    it('creates a request handler', () => {
       mockRouter = new MockRouter({
         method: 'get',
         path: '/api/workplace_search/groups/{id}',
-        payload: 'params',
```

Comment:
FYI - I made this change for App Search too, but we're basically never validating URL params since they're so straightforward (always strings/always present) so I started removing them as a payload option since we aren't using it to validate on